### PR TITLE
Update intercom.js license format to SPDX

### PR DESCRIPTION
**Pull request by programmiri**
Pull request for issue: #11931

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Squashing into only one commit now.

File: ajax/libs/equalize.js/package.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
   ],
   "homepage": "https://github.com/tsvensen/equalize.js",
   "licenses": [
-    "MIT",
+    {
```
Comment (maruilian11) [inline ajax/libs/equalize.js/package.json]: Please use `GPL-2.0` rather than modify `MIT` part. Thanks.

Comment (programmiri) [reply ajax/libs/equalize.js/package.json]: @maruilian11 I think that got fixed in another branch of mine (equalize.js) https://github.com/cdnjs/cdnjs/pull/11988  I'm not sure why it's popping up in the branch intercom.js :thinking:

Comment (PeterDaveHello) [reply ajax/libs/equalize.js/package.json]: Looks like you use a wrong branch(#11988) as base to create another branch for this PR, let's just change it back here, and we'll help you squash it.

Comment (programmiri) [reply ajax/libs/equalize.js/package.json]: ok, done :)

File: ajax/libs/equalize.js/package.json [hunk 2]
```diff
@@ -14,10 +14,7 @@
   "homepage": "https://github.com/tsvensen/equalize.js",
   "licenses": [
     "MIT",
-    {
-      "type": "GPLv2",
-      "url": "http://opensource.org/licenses/GPL-2.0"
-    }
+    "GPL-2.0"
```
Comment (PeterDaveHello) [inline ajax/libs/equalize.js/package.json]: This change should be reverted.

File: ajax/libs/equalize.js/package.json (hunk 1 above)
Comment (PeterDaveHello) [reply ajax/libs/equalize.js/package.json]: hmm ... looks not correct, @programmiri I think you should try to make the "diff" disappeared here in https://github.com/cdnjs/cdnjs/pull/11991/files

**Comment by PeterDaveHello** [comment]
@programmiri I think we are almost there, can you please try to make the number of "files changed" here: https://github.com/cdnjs/cdnjs/pull/11991/files only 1? Then we can merge this, thanks!

**Comment by programmiri** [comment]
@PeterDaveHello Sorry, I'm not quite sure how to do this. Should I just reverse my changes at the file equalize.js/package.json file because it's the wrong branch anyways?

**Comment by PeterDaveHello** [comment]
@programmiri Yes, reverse until you see there is only one file been modified on the page :)

**Comment by PeterDaveHello** [comment]
@programmiri 👍

**Pull request merged**